lib/data-types.js

```diff
@@ -503,6 +503,22 @@ DATEONLY.prototype.toSql = function() {
 };
 
 /**
+ * A timestamp column. Only available in mssql.
```

Comment:
Will it be better to use Sequelize.ROWVERSION instead of Sequelize.TIMESTAMP? I think some projects are using TIMESTAMP as well, but it's for other dialects, and that data type can be modified unlike in MSSQL. [Example for MySQL](https://github.com/sequelize/sequelize/issues/3989)